lib/krikri.rb

```diff
@@ -10,6 +13,8 @@ module Krikri
   autoload :Mapping,        'krikri/mapping'
   autoload :MappingDSL,     'krikri/mapping_dsl'
   autoload :Parser,         'krikri/parser'
+  autoload :ValidationReportList, 'krikri/validation_report_list'
```

Comment:
These two lines probably won't  be necessary after rebase. `./lib/*` is now being autoloaded in bulk.